projects/angular/src/emphasis/_labels.clarity.scss

```diff
@@ -110,84 +47,49 @@
   .label,
   a.label {
     //default
-    $label-nudge-for-vertical-alignment: variables.$clr_baselineRem_1px;
-    @include mixins.css-var(
-      font-size,
-      clr-label-font-size,
-      label-variables.$clr-label-font-size,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      font-weight,
-      clr-label-font-weight,
-      label-variables.$clr-label-font-weight,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      letter-spacing,
-      clr-label-letter-spacing,
-      label-variables.$clr-label-letter-spacing,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      line-height,
-      clr-label-line-height,
-      // line-height needed to be adjusted to force center alignment
-      label-variables.$clr-label-font-size + variables.$clr_baselineRem_1px,
-      variables.$clr-use-custom-properties
-    );
+    @include mixins.generate-typography-token('CAPTION-LG-11');
+    // TODO: Check why there is discrepancy between spec and previous styling for letter-spacing and line-height
+    letter-spacing: var(--clr-label-letter-spacing, 0.03em);
```

Comment:
I am only adding these CSS rules so that we don't have visual diffs

Without them the diffs are the following: https://github.com/vmware-clarity/ng-clarity/commit/55a7cba889e61fffbe4e9b28a70fc252c501db8b

Reply:
The spec clearly shows the `line-spacing` from `CAPTION-LG-11` is the correct one.
Visual changes are desired in this situation.